metadata_event_handler/consumer.py

```diff
@@ -99,6 +99,7 @@ def decode_message(self, message: bytes, topic: str) -> dict[str, Any]:
                 message_decoded = schemaless_reader(message_reader, imported_schema)
                 # prevent writers schema changes to be promotable to readers schema
                 is_valid = validate(message_decoded, imported_schema, raise_errors=False)
+                logger.info(f'Decoded a message from {topic}: {message_decoded}')
```

Comment:
i think using the extra is better.
```suggestion
                logger.info('Decoded a message:', extra={'topic': topic, 'msg': message_decoded})
```